### DISCUSSION TITLE
pimcore docker instructions / FAST installation path

### DISCUSSION THREAD

**Discussion by spersels** [pimcore/pimcore#7711] [Q&A / Support] [answered]
Looking to get a local instance of pimcore running and thought that spinning up a docker container would be the easiest path.

Yes, I could go through the full system setup and installation but I didn't want to do that until I'd worked with the application a bit was sure it was suitable for my needs.

But the docker images that are available are a mystery as to how they work.  I've pull a couple of the images and spun them up but none of them seem to leave me with a running system.

Can someone help here?  I just need a copy of the SW running that I can work with for a while - primarily a functional checkout

Thanks,

Shawn

**Comment by mgomse** [comment, marked as answer]
Good morning @spersels ,

I think that https://github.com/dpfaffenbauer/pimcore-docker-compose can be very useful for you - you can find clear instructions in the README.

**Comment by spersels** [comment]
Thank you very much !  Exactly what I needed.

**Reply by spersels** [reply]
Thanks for your help - everything seemed to work with a couple of exceptions:
Running composer I saw stuff like this

![image](https://user-images.githubusercontent.com/6519929/101827211-0088c680-3afe-11eb-93b5-3d91af8aed59.png)

Running the installation after COMPOSE seemed to be ok.

![image](https://user-images.githubusercontent.com/6519929/101827491-62e1c700-3afe-11eb-80d2-c73b1cef3096.png)

But then connecting to the app resulted in:

![image](https://user-images.githubusercontent.com/6519929/101827595-83aa1c80-3afe-11eb-859f-82acdb109d4f.png)

Any thoughts?

**Reply by spersels** [reply]
Never  mind - got it.

My problem was I was using windows as a host and the docker-compose was creating a local volume connection with the container.   I did manage to get it running that way but performance was DEAD SLOW.   I removed the local volume and installed the SW directly on the container.    All the errors went away and performance is now very snappy.

THanks!
